Makefile.include

```diff
@@ -143,10 +143,13 @@ include $(RIOTMAKE)/info-nproc.inc.mk
 # List of boards variables
 include $(RIOTMAKE)/boards.inc.mk
 
+<<<<<<< HEAD
```

Comment:
Missing from rebase.

Reply:
🤦‍♂

Reply:
oh man, I need ☕️ ... this took longer than expected. However, should be good and working now.